cdi/4.0/_index.md

```diff
@@ -21,30 +42,17 @@ The Jakarta Contexts and Dependency Injection 4.0 goals for this release are:
     * [All issues with a Lite label](https://github.com/eclipse-ee4j/cdi/issues?q=is%3Aissue+label%3ALite)
 
 
-* [Jakarta Contexts Dependency Injection 4.0 Release Record](https://projects.eclipse.org/projects/ee4j.cdi/releases/4.0)
-<!-- 
-* [Jakarta Contexts Dependency Injection 4.0 Specification Document]() (PDF)
-* [Jakarta Contexts Dependency Injection 4.0 Specification Document]() (HTML)
-* [Jakarta Contexts Dependency Injection 4.0 Javadoc]()
-* [Jakarta Contexts Dependency Injection 4.0 TCK]()
-([sig](),
-[sha](),
-[pub]()
-* Maven coordinates
-  * [jakarta.enterprise:jakarta.cdi-api:jar:4.0.0]()
-* Schemas
-  * [XML Schema for the CDI 4.0 beans.xml deployment](https://jakarta.ee/xml/ns/jakartaee/beans_4_0.xsd)
--->
-
-
-# Compatible Implementations
-
-* TBD
 
 # Ballots
 
 ## Plan Review
 
+This Specification Project's Plan Review was covered by the [CDI 4.0 Plan Review](https://projects.eclipse.org/projects/ee4j.cdi/releases/4.0/plan).  
+Please reference that ballot for the official results.
+
+
+## Release Review
+=======
```

Comment:
I suggest to delete this line as explained above.

Reply:
The preview looks ok now.